### PR TITLE
Test for more specific case match first

### PR DESCRIPTION
**Pull request by dirkmueller**
Otherwise the first match wins and the more specific one (.*:Ports)
is never selected.

### CODE REVIEW

File: doit.sh [hunk 1]
```diff
@@ -14,12 +14,12 @@ case $proj in
                 repo="standard"
                 product=_product
                 ;;
-        Leap:15.*) arches="x86_64"
-                repo="standard"
+        Leap:15.*:Ports) arches="aarch64"
+                repo="ports"
```
Comment (michelmno) [inline doit.sh]: Why not to keep ppc64le ?

**Comment by ggardet** [comment]
@DimStar77 Could review and merge, please?

**Comment by dirkmueller** [comment]
@coolo can you help here?

**Comment by coolo** [comment]
Nope, this is the obsolete packagelist generator. I haven't touched this in years.

**Comment by nilxam** [comment]
I'm fine with that, but I'd like to put an clear information here,

*Tumbleweed* still use package-lists(ie. doitall), however for Leap, *Leap* already changed to https://github.com/openSUSE/openSUSE-release-tools/blob/master/pkglistgen.py , in any case you'd like to generate packagelist for Leap, I suggest you use pkglistgen.py on Leap ports, just for consistency.

**Pull request merged**